vxtelegram/telegram.py

```diff
@@ -187,12 +177,19 @@ def handle_raw_inbound_message(self, message_id, request):
             message_id=message_id,
             content=message['content'],
             to_addr=message['to_addr'],
+            to_addr_type='telegram_username',
             from_addr=message['from_addr'],
+            from_addr_type='telegram_id',
```

Comment:
Is there any way to look up the `telegram_username` for the `telegram_id`? It would be nice to be able to consistently use `telegram_username` for both `to_addr` and `from_addr` (not for this PR though).